charts/orchestrator/values.yaml

```diff
@@ -69,6 +69,12 @@ rhdhPlugins: # RHDH plugins required for the Orchestrator
   notifications_email:
     package: "plugin-notifications-backend-module-email-dynamic@0.0.0-rc.0-0"
     integrity: sha512-TikxFBxBHKJYZy8go+Mw+7yjfSJILgXjr4K0C0+tnKyMOn+OqIX6K8c1fq7IdXto3fftQ+mmCrBqJem25JjVnA==
+    hostname: "" # SMTP server hostname
+    port: 587 # SMTP server port
+    username: "" # SMTP username
+    password: "" # SMTP password
```

Comment:
Pls replace this one with a value from the secret and add this as an argument to the setup.sh script.